cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/AnomalyDetection.scala

```diff
@@ -1,9 +1,10 @@
 // Copyright (C) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in project root for information.
 
-package com.microsoft.azure.synapse.ml.cognitive
+package com.microsoft.azure.synapse.ml.cognitive.anomaly
```

Comment:
aren't these namespace changes breaking changes?

Reply:
For scala yes, we need to change namespaces when importing them. But for python, I'll make it backward compatible.